src/components/DeveloperUtility/DeveloperUtility.module.scss

```diff
@@ -1,25 +1,49 @@
 .container {
-  background: black;
-  color: white;
-  position: fixed;
-  top: calc(var(--spacing-mega) + var(--spacing-medium));
-  right: calc(var(--spacing-mega) + var(--spacing-medium));
-  height: calc(2*var(--spacing-mega));
-  width: calc(2*var(--spacing-mega));
-  
-  z-index: var(--z-index-max);
-  border-radius: 100%;
-  box-shadow: var(--shadow-regular);
-  text-align: center;
+	background: rgb(230, 230, 230);
```

Comment:
Thank you microArt for your PR! Love the new functionality.

We avoid using hardcoded values for colors, sizing etc. Please check the theme.scss file and see which variable most closely aligns with your desired UI. 

It applies for all the custom values in this PR.